### PR TITLE
[14.0][MIG] sale_order_line_date

### PR DESCRIPTION
**Pull request by cor9003**
Standard migration and adapt testing to codecov range

### CODE REVIEW

**Comment by cor9003** [comment]
Hi, all green. Do I have to do any step for review ?. Thanks

**Comment by netfxtech** [comment]
Hi cor, will you please confirm your stock pickings are generating correctly? I'm still only receiving one delivery order 
image

**Comment by cor9003** [comment]
Hi,

The objective of this module is not split stock pickings based in 
different dates, only to inform dates in every stock move and also in 
sale order line and commintmet date.

The module sale_deilvery_split_date is in charge of that changing the 
procurement, but before migrated the sale_deilvery_split_date is 
necessary migrate it because one depends of other

By the way I' found and error in my previuos commit has been solved now 
in the date of stock move :-)

Thanks



DMPyme <http://www.dmpyme.com>

David Martinez Garcia / Consultor
david.martinez@dmpyme.com <mailto:david.martinez@dmpyme.com>

DMPyme
93 676 34 18 Ext - 101
C/ Major 26 Sant Climent de Llobregat, Barcelona
www.dmpyme.com <http://www.dmpyme.com>

**Review by danimv5 — APPROVED** [review]
Functional review. LGTM!!

File: sale_order_line_date/models/sale_order_line.py [hunk 1]
```diff
@@ -27,13 +27,6 @@ def write(self, vals):
                     break
         return super(SaleOrderLine, self).write(vals)
 
-    @api.model
-    def create(self, vals):
-        res = super(SaleOrderLine, self).create(vals)
-        if res.order_id.commitment_date and not res.commitment_date:
-            res.write({"commitment_date": res.order_id.commitment_date})
-        return res
-
```
Comment (CasVissers-360ERP) [inline sale_order_line_date/models/sale_order_line.py]: @cor9003 Why did you remove this method?

**Review by CasVissers-360ERP — COMMENTED** [review]
Code review question

File: sale_order_line_date/models/sale_order_line.py (hunk 1 above)
Comment (cor9003) [reply sale_order_line_date/models/sale_order_line.py]: Hello, we were done several testing and this code never had this casuistic res.order_id.commitment_date and not res.commitment_date, for this reason never should be executed.
Checked the test data in the module we saw that never pass through this code neither and codecov also confirmed it.

**Review by CasVissers-360ERP — APPROVED** [review]

**Comment by rousseldenis** [comment]
@cor9003 Is this ready ?

**Comment by cor9003** [comment]
@rousseldenis Yes, ready. Do i have any additional thing before merge ?

**Comment by rousseldenis** [comment]
> @rousseldenis Yes, ready. Do i have any additional thing before merge ?

Nope, just to check if last commit is significant by its own or needs to be squashed.

**Comment by cor9003** [comment]
> > @rousseldenis Yes, ready. Do i have any additional thing before merge ?
> 
> Nope, just to check if last commit is significant by its own or needs to be squashed.

From my point of view es Ok

**Review by rousseldenis — APPROVED** [review]
